setup/ubuntu14.04/ansible-vars.yaml

```diff
@@ -15,4 +15,3 @@ packages:
   - libtool
   - curl
   - ntp
-  - ccache
```

Comment:
I've rarely run into issues with ccache, especially versions `>=3`. If that's provided I don't see the point in using our hand baked stuff.

One benefit _could_ be consistent paths to the compiler symlinks, but since most init scripts vary per os it's of less significance.